app/src/main/java/com/wtmberlin/data/WtmEventDao.kt

```diff
@@ -4,15 +4,14 @@ import androidx.room.Dao
 import androidx.room.Insert
 import androidx.room.Query
 import androidx.room.Transaction
-import io.reactivex.Flowable
 
 @Dao
 interface WtmEventDao {
     @Query("SELECT * FROM WtmEvent")
-    fun getAll(): Flowable<List<WtmEvent>>
+    suspend fun getAll(): List<WtmEvent>
 
     @Query("SELECT * FROM WtmEvent WHERE id = :eventId")
-    fun getById(eventId: String): Flowable<WtmEvent>
+    suspend fun getById(eventId: String): WtmEvent
```

Comment:
You should use an optional here, to avoid crashing the app if the item doesn't exist.
```suggestion
    suspend fun getById(eventId: String): WtmEvent?
```

Reply:
Ok. I'll add